src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -412,6 +415,7 @@ protected void registerRecipes(Consumer<IFinishedRecipe> recipeSaver)
 		
 		GristCostRecipeBuilder.of(MSItems.CAPTCHAROID_CAMERA).grist(BUILD, 5000).grist(CAULK, 500).grist(GOLD, 500).grist(MARBLE, 500).grist(MERCURY, 500).grist(SHALE, 500).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSBlocks.TRANSPORTALIZER).grist(BUILD, 350).grist(AMETHYST, 27).grist(RUST, 36).grist(URANIUM, 18).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSBlocks.TRANS_PORTALIZER).grist(BUILD, 450).grist(RUBY, 11).grist(COBALT, 20).build(recipeSaver);
```

Comment:
With the crafting recipe for the trans portalizer, I'd rather you skip the manual grist cost and let it use the autogenerated cost instead (so remove either the grist cost or the crafting recipe)